portal/models/intervention_strategies.py

```diff
@@ -286,14 +286,14 @@ def intro_html(assessment_status):
                         u"Please complete your {} "
```

Comment:
Tiny detail here, but if no `assigning_authority` is found, won't these strings read as "Please complete your  questionnaire" (in other words, have two consecutive spaces)?